### PR TITLE
Revisit log levels

### PR DESCRIPTION
**Pull request by ClifHouck**

### CODE REVIEW

**Comment by jayofdoom** [comment]
Seems reasonable, if tests pass. +1

File: arsenal/director/onmetal_scout.py [hunk 1]
```diff
@@ -181,9 +181,9 @@ def _find_glance_image(self, cache_node_action):
         return None
 
     def issue_cache_node(self, cache_node_action):
-        LOG.debug("Issuing cache node operation on node %(node)s with "
-                  "image %(image)s", {'node': cache_node_action.node_uuid,
-                                      'image': cache_node_action.image_uuid})
+        LOG.info("Issuing cache node operation on node %(node)s with "
+                 "image %(image)s", {'node': cache_node_action.node_uuid,
```
Comment (slick666) [inline arsenal/director/onmetal_scout.py]: wouldn't it be safer to use the getattr function here to gracefully handle malformed data?
```
getattr(cache_node_action, 'node_uuid', None)
```

File: arsenal/external/client_wrapper.py [hunk 2]
```diff
@@ -169,11 +169,11 @@ def call(self, method_name, *args, **kwargs):
...
                 self._invalidate_cached_client()
-                LOG.debug("The wrapped %(name)s client became unauthorized. "
-                          "Will attempt to reauthorize and try again." % {
-                              'name': self.name})
+                LOG.info("The wrapped %(name)s client became unauthorized. "
+                         "Will attempt to reauthorize and try again." %
+                         {'name': self.name})
             except retry_exceptions as e:
-                LOG.debug("Got a retry-able exception." + str(e))
+                LOG.info("Got a retry-able exception." + str(e))
```
Comment (slick666) [inline arsenal/external/client_wrapper.py]: Wouldn't log.exception be more appropriate here?

File: arsenal/director/onmetal_scout.py (hunk 1 above)
Comment (ClifHouck) [inline arsenal/director/onmetal_scout.py]: Probably. Though, to get to this point the action's class has been identified by isinstance, and the class in question sets that property as part of the __init__ process. See https://github.com/rackerlabs/arsenal/blob/master/arsenal/strategy/base.py#L122

File: arsenal/external/client_wrapper.py (hunk 2 above)
Comment (ClifHouck) [inline arsenal/external/client_wrapper.py]: Yes. I will change this.

**Pull request merged**